lightning/src/ln/features.rs

```diff
@@ -251,28 +265,34 @@ mod sealed {
 					const ASSERT_ODD_BIT_PARITY: usize = (<Self as $feature>::ODD_BIT % 2) - 1;
 				}
 			)*
+
 		}
 	}
 
 	define_feature!(1, DataLossProtect, [InitContext, NodeContext],
-		"Feature flags for `option_data_loss_protect`.");
+		"Feature flags for `option_data_loss_protect`.", set_supports_data_loss_protect,
+		set_requires_data_loss_protect);
```

Comment:
It kinda sucks that we have to pass separate names for these. Especially considering we'd have to pass two more if we want to be consistent and define the existing `supports_` and `requires_` predicates using macros, too. Not sure if there is a better solution, though.

Reply:
I think technically you could do a procedural macro for this, but I don't know how to use them and I don't really want to find out.